chat-server.py

```diff
@@ -0,0 +1,84 @@
+import socket
+import selectors
+
+HEADER_LENGTH = 10
+HOST = '127.0.0.1'
```

Comment:
Use 0.0.0.0 to bind to "any", i.e. allow clients to connect via any network interface

Reply:
Ok